src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,67 @@
+package duke;
+
+import duke.command.DukeCommandWithArgs;
+import duke.exception.DukeStorageException;
+import duke.exception.InvalidCommandException;
+import duke.task.TaskList;
+
+public class Duke {
+    private static final String SAVE_FILE_LOCATION = "duke-task-list.txt";
+    private final Ui ui;
+    private final TaskList taskList;
+    private final Storage storage;
+    private final Parser parser;
+
+    /**
+     * Duke constructor
+     */
+    public Duke() {
+        this.ui = new Ui(System.in, System.out);
+        this.storage = new Storage(SAVE_FILE_LOCATION);
+        TaskList taskList;
+        try {
+            taskList = storage.loadTaskList();
+        } catch (DukeStorageException e) {
+            ui.outputLine(e.getMessage());
+            taskList = new TaskList();
+        }
+        this.taskList = taskList;
+        this.parser = new Parser();
+    }
+
+    /**
+     * Runs Duke.
+     * @param args The arguments passed in from the command line
+     */
```

Comment:
Should there be an empty line between the description and parameter section?
It is mentioned in the coding standard, and I noticed this multiple times